demo/todo.js

```diff
@@ -38,7 +38,7 @@ class TodoItems extends Component {
 	render({ todos, removeTodo }) {
 		return todos.map( todo => (
 			<li key={todo.id}>
-				<button onClick={removeTodo} data-id={todo.id}>&times;</button>
+				<button type="button" onClick={removeTodo} data-id={todo.id}>&times;</button>
```

Comment:
Oh I actually thought this only happened inside of `<form>`

Reply:
You're right it does! The `TodoList` component adds a `<form>` element around the items https://github.com/developit/preact/blob/efb56776618d7fa62929776421c68dfd6681a54d/demo/todo.js#L25-L31